README-translated/README-Italian.md

```diff
@@ -24,38 +50,49 @@ Indice:
-Questo progetto è sviluppato in maniera aperta e nella nostra repository github sono benvenuti tutti i suggerimenti, le domande e i commenti: <https://github.com/alan-turing-institute/the-turing-way>.
+Questo progetto è sviluppato in maniera aperta e nel nostro repository GitHub sono benvenuti suggerimenti, domande e commenti: <https://github.com/alan-turing-institute/the-turing-way>.
 
 ### Il team
 
-Questi sono i membri del team (solo alcuni) al lavoro al Turing Institute.
-Per i dettagli su come contattarci, visita [ways of working document](ways_of_working.md).
+_The Turing Way_ è una collaborazione aperta e guidata dalla comunità. Chiunque partecipi a questo libro, indipendentemente dall'importanza del contributo, è riconosciuto in questo progetto come contributore e membro della comunità. Collaboratori di lunga data sono considerati come collaboratori principali ed hanno ruoli di leadership all'interno del progetto.
+Il progetto è coordinato da **Kirstie Whitaker** (founder) e **Malvika Sharan** (community developer) ed è ospitato dal [The Alan Turing Institute](https://www.turing.ac.uk/).
 
-![Team photo](book/website/figures/TuringWayTeam.jpg)
+Per maggiori informazioni, puoi leggere [questa guida](https://the-turing-way.netlify.app/community-handbook/acknowledgement/acknowledgement-members.html#community-members-contributors-and-co-authors) e la [Lista dei Contributori](https://the-turing-way.netlify.app/afterword/contributors-record.html) per capire come riconosciamo il tuo lavoro e come i nostri contributori sono messi in evidenza nel progetto.
+
+Consulta [l'elenco dei collaboratori](#contributors) per i profili GitHub di tutti i contributori.
 
 ### Come contribuire
 
-:construction: Questa repository è sempre in costruzione e **tutti** sono incoraggiati ad aiutarci in questo progetto che può essere utile a molti. :construction:
+:construction: Questo repository è sempre in costruzione e **tutti** sono incoraggiati ad aiutarci in questo progetto che può essere utile a molti. :construction:
+
+Tutti sono tenuti a seguire il nostro [codice di condotta](CODE_OF_CONDUCT.md) e a leggere le nostre [linee guida](CONTRIBUTING.md) per avere più informazioni su come iniziare.
+Vogliamo rendere la contribuzione a questo progetto il più facile possibile.
+Per questo motivo, ci sono diversi modi in cui è possibile contribuire in base ai propri interessi, disponibilità e capacità.
+
+![Questa immagine mostra sei dei diversi modi in cui chiunque può contribuire. Questi sono: sviluppare e condividere, mantenere e migliorare, condividere risorse, revisione e aggiornamento, rendere il progetto globale attraverso la traduzione, e condividere le migliori pratiche.](https://github.com/alan-turing-institute/the-turing-way/blob/master/book/website/figures/README_imgs/README_contributions.png)
```

Comment:
given that all other are verbs at the infinite tense, you could have "revisionare e aggiornare"?